automatic/kubescape/kubescape.nuspec

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>kubescape</id>
+    <version>2.3.3</version>
+    <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/kubescape</packageSourceUrl>
+    <owners>chocolatey-community</owners>
+    <title>kubescape: Kubernetes Security Platform</title>
+    <authors>Kubescape authors</authors>
+    <projectUrl>https://www.armosec.io/kubescape/</projectUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@master/icons/kubescape.png</iconUrl>
+    <licenseUrl>https://github.com/kubescape/kubescape/blob/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/kubescape/kubescape</projectSourceUrl>
+    <docsUrl>https://hub.armosec.io/docs</docsUrl>
+    <bugTrackerUrl>https://github.com/kubescape/kubescape/issues</bugTrackerUrl>
+    <tags>kubescape kubernetes security devops vulnerability-detection nsa best-practice mitre-attack</tags>
```

Comment:
Please update the second tag to be one of of the following tags (convention that we want to get started).

- `addon` (for packaged software that is a plugin or addon for another application)
- `browser` (For browser implementations like Google Chrome, Firefox, Microsoft Edge, Brave, and Opera.)
- `client`
- `driver`
- `editor`
- `extension` (only for Chocolatey Extensions)
- `games`
- `productivity`
- `programming`
- `server`
- `utility`
- `web`

and the third tag to be one of

- `foss` (The packaged application is a free application with its source freely viewable by anyone)
- `freeware` (The packaged application is a free application, but its source is not available to be viewed)
- `oss` (The packaged application has its source freely viewable by anyone but requires payment for compiled binaries)
- `trial` (The packaged application is a commercial or paid application but includes a limited free trial)

If the application is also cross platform, add the tag `cross-platform` as the fourth tag.